File: spring-boot-java/src/main/java/io/ssaragi/java/spring/boot/dto/data/exam/ExamData.java
package io.ssaragi.java.spring.boot.dto.data.exam;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.math.BigDecimal;

@Getter
@Setter
@ToString
public class ExamData {

    private String valStr;
    private Integer valInt;
    private Long valLong;
    private BigDecimal valDecimal;
}

File: spring-boot-kotlin/src/main/kotlin/io/ssaragi/kotlin/spring/boot/dto/data/exam/ExamData.kt
package io.ssaragi.kotlin.spring.boot.dto.data.exam

import java.math.BigDecimal

data class ExamData(

    var valStr: String? = null,
    var valInt: Int? = null,
    var valLong: Long? = null,
    var valDecimal: BigDecimal? = null,
)
